frontend/expr_inferrer.py

```diff
@@ -375,33 +376,74 @@ def _get_args_types(args, context, instance, solver):
     return args_types
 
 
-def _infer_annotated_function_call(args_types, solver, annotations):
-    result_type = solver.new_z3_const("call")
-    solver.annotation_resolver.add_annotated_function_axioms(args_types, solver, annotations, result_type)
+def _infer_annotated_function_call(args_types, solver, annotations, result_type):
+    return solver.annotation_resolver.get_annotated_function_axioms(args_types, solver, annotations, result_type)
 
-    return result_type
+
+def _infer_builtin_method(args_types, solver, context, result_type, method_name):
+    """Get the axioms of built-in methods inference"""
```

Comment:
I find the naming (and the docstring) a bit weird here. We're not inferring the type of a builtin method here, right? We're collecting constraints for a call for the case that it refers to a builtin method.